src/backend/seeds/05_groups.js

```diff
@@ -0,0 +1,59 @@
+const { ENV_IS_STAGING_OR_PROD } = require('../knexfile');
+const { parseTimeFromRequest } = require('../utils/db-dates.js');
+
+// An issue with deploying is that we require this seeding for the functionality on the actual deployment.
+// A new table that is deployed needs to be migrated the first time its deployed, then, in order to avoid this, we would put the !ENV_IS_STAGING_OR_PROD checker again?
+
+if (!ENV_IS_STAGING_OR_PROD) {
```

Comment:
The if-else statements both do the same thing, so we can remove the if statement.